packages/realm-server/main.ts

```diff
@@ -43,6 +44,38 @@ if (!REALM_SECRET_SEED) {
   process.exit(-1);
 }
 
+const REALM_SERVER_SECRET_SEED = process.env.REALM_SERVER_SECRET_SEED;
```

Comment:
it might be easier to just use the REALM_SECRET_SEED instead of making a new one. since these are encrypted values in our SSM, i would guess that if one could be compromised all of them could be compromised. so making separate ones seems like more of a hassle to me.